src/app.py

```diff
@@ -14,15 +16,88 @@
 app = FastAPI(debug=DEBUG,
               title='Linktious')
 
+
+# Dependency
+def get_db():
+    db = base.SessionLocal()
+    try:
+        yield db
+    finally:
+        db.close()
+
+
+# TODO: add validation to routes
+# TODO: return status codes
 @app.get("/")
 def read_root():
     return {"status": "OK"}
 
+
 @app.get("/health")
 def health_check():
     return {"status": "Ok"}
 
 
+@app.get("/teams", response_model=List[schema.Team])
+def get_team(db: Session = Depends(get_db)):
+    return crud.get_teams(db=db)
+
+
+@app.get("/users/{user_email}", response_model=schema.User)
+def get_user_by_email(user_email: str, db: Session = Depends(get_db)):
+    return crud.get_user_by_email(db=db, user_email=user_email)
+
+
+@app.get("/users/{user_id}", response_model=schema.UserBasicInfo)
+def get_user_basic_info_by_id(user_id: int, db: Session = Depends(get_db)):
+    return crud.get_user_by_id(db=db, user_id=user_id)

```

Comment:
I decided to create two different serializers because one is for request from the user itself (including its favorite boards and main id) and another one is user A requesting info about user B.

Reply:
So I dont think it should be with email.
I think a better choice is 
/users/self
or something like that
allowing to query by mail is IMO the same as id semantically

Reply:
Yeah I agree
I will change this method to `login` which will be dummy method for now and later on when we add real authentication we can update it.
What do you think?

Reply:
no objections